libraries/plugins/witness/witness.cpp

```diff
@@ -109,6 +109,16 @@ void witness_plugin::plugin_initialize(const boost::program_options::variables_m
          _private_keys[key_id_to_wif_pair.first] = *private_key;
       }
    }
+   if(options.count("required-participation"))
+   {
+       auto required_participation = options["required-participation"].as<uint32_t>();
+       FC_ASSERT(required_participation < 100);
+       _required_witness_participation = options["required-participation"].as<uint32_t>()*GRAPHENE_1_PERCENT;
+       if(required_participation < 10)
+           ilog("witness plugin: Warning - Low required participation");
+       else if(_required_witness_participation > 90)
```

Comment:
`90` for `_required_witness_participation` means `0.9%`.

Reply:
my apologies for that, fixing with the other suggestions.

Reply:
fixed at https://github.com/bitshares/bitshares-core/pull/1787/commits/0bdf4d6d2b53305b05f11a0493162bc27a0bd49a

Reply:
changed at https://github.com/bitshares/bitshares-core/pull/1787/commits/0bdf4d6d2b53305b05f11a0493162bc27a0bd49a